frontend/src/entry/overview/Overview.tsx

```diff
@@ -13,18 +14,24 @@ type OverviewPageKeys = (typeof OverviewPage)[keyof typeof OverviewPage]
 
 type OverviewProps = {
   entry: EntryInterface
+  readOnly?: boolean
 }
 
-export default function Overview({ entry }: OverviewProps) {
+export default function Overview({ entry, readOnly = false }: OverviewProps) {
   const page: OverviewPageKeys = useMemo(
     () => (entry.card && entry.card.schemaId ? OverviewPage.FORM : OverviewPage.TEMPLATE),
     [entry.card],
   )
 
-  return (
+  return readOnly ? (
+    <MessageAlert
+      severity='warning'
+      message='This mirrored model has no model card. Please export the model card from the source model.'
+    />
+  ) : (
     <Container sx={{ my: 2 }}>
       {page === OverviewPage.TEMPLATE && <TemplatePage entry={entry} />}
-      {page === OverviewPage.FORM && <FormEditPage entry={entry} />}
+      {page === OverviewPage.FORM && <FormEditPage entry={entry} readOnly />}
```

Comment:
Why are you passing `readOnly` into `FormEditPage`? Based on your logic `FormEditPage` will not be rendered in read-only mode, so the `readOnly` prop will always be `false`.